src/components/MangaControl.tsx

```diff
@@ -35,7 +41,7 @@ type Props = {
 }
 
 const MangaControl: React.FunctionComponent<Props> = (props: Props) => {
-  const { readerSettings, updateReaderSettings, chapter, chapters, updateChapter, page, pages, updatePage } = props;
+  const { readerSettings, updateReaderSettings, keyBindings, setKeyBindings, chapter, chapters, updateChapter, page, pages, updatePage } = props;
   const toggleMenuState = () => {
     updateReaderSettings({ ...readerSettings, menu: readerSettings.menu === Menu.open ? Menu.close : Menu.open, });

```

Comment:
also is readerSettings.menu ever going to have more states than just open or close?

if not you could just change it to menuOpen or menuIsOpen and have it be a bool